hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1751,6 +1752,7 @@ message CancelSnapshotDiffRequest {
   optional string bucketName = 2;
   optional string fromSnapshot = 3;
   optional string toSnapshot = 4;
+  optional bool cancel = 8;
```

Comment:
this change do not seems related to this PR, please check

Reply:
Thank you @sumitagrawl! Removed the field, looks like it has been added by mistake during merge.